Directory.Build.targets

```diff
@@ -5,11 +5,16 @@
 
   <!-- 
     Repeating this here because the Arcade VisualStudio.props runs after our Versions.props and overwrites
-    our VsixVersion
+    our VsixVersion.
+
+    If this is developer build - we need to have a LARGER number than our official builds, which will
+    look like: 16.0.20181207.3
+
+    The current scheme should work until the 43rd century.
   -->
   <PropertyGroup>
     <VsixVersion>16.0</VsixVersion>
     <VsixVersion Condition="'$(OfficialBuildId)' != ''">$(VsixVersion).$(OfficialBuildId)</VsixVersion>
-    <VsixVersion Condition="'$(OfficialBuildId)' == ''">$(VsixVersion).4242424</VsixVersion>
+    <VsixVersion Condition="'$(OfficialBuildId)' == ''">$(VsixVersion).42424242.42</VsixVersion>
```

Comment:
I realize that there is duplication. Right now this has the set of workarounds that we need to fix versioning in all of the places that we care about. I'm going to reach out to the right people about how we remove these workarounds later.